package.json

```diff
@@ -38,6 +38,7 @@
     "@civic/solana-gateway-react": "0.11.0",
     "@dialectlabs/react-sdk-blockchain-solana": "1.0.0-beta.3",
     "@dialectlabs/react-ui": "1.1.0-beta.5",
+    "@dual-finance/staking-options": "^0.0.3",
```

Comment:
Would it be possible to target a specific version of the sdk?

Latest is `0.0.7` 

https://www.npmjs.com/package/@dual-finance/staking-options